src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -181,6 +187,30 @@ private IPackageInstallerService GetPackageInstallerService(Document document)
             return fuzzyReferences;
         }
 
+        private ISymbolSearchService GetSymbolSearchService(Document document) 
+            => document.Project.Solution.Options.GetOption(SymbolSearchOptions.SuggestForTypesInReferenceAssemblies, document.Project.Language)
+                ? _symbolSearchService ?? document.Project.Solution.Workspace.Services.GetService<ISymbolSearchService>()
+                : null;
+
+        private IPackageInstallerService GetPackageInstallerService(Document document)
+            => document.Project.Solution.Options.GetOption(SymbolSearchOptions.SuggestForTypesInNuGetPackages, document.Project.Language)
+                ? _packageInstallerService ?? document.Project.Solution.Workspace.Services.GetService<IPackageInstallerService>()
+                : null;
+
+        private ImmutableArray<PackageSource> GetPackageSources(Document document, ISymbolSearchService searchService)
+        {
+            if (searchService != null)
```

Comment:
It is... but i guess i'm saying "if we can't even search a nuget package, there's no point having a list of package sources to look at.  I can move that higher if you want.